docker-compose.yml

```diff
@@ -10,22 +10,128 @@ services:
   eigenlayer:
     platform: linux/amd64
     image: ghcr.io/breadchaincoop/eigenlayer:dev
-    depends_on:
```

Comment:
why remove this?

Reply:
mistake, amended with 46238d7e53e8b9af9b81aa75f3c08759694b3471